order/cache.go

```diff
@@ -0,0 +1,110 @@
+package order
+
+import (
+	"bytes"
+	"encoding/gob"
+	"io/ioutil"
+	"log"
+	"time"
+
+	"github.com/allegro/bigcache"
+	"github.com/foomo/shop/order"
+	"github.com/foomo/shop/state"
+	"github.com/foomo/shop/version"
+)
+
+var cache *bigcache.BigCache
+
+func init() {
+
+	log.Println("init order cache")
```

Comment:
This log entry would be sucky, since it would break the JSON formatter in apps

Reply:
true. was just for debugging purposes to check how often and when the cache is initialised.